Assertion Faled

The MyCrypto desktop version crashes with a full red screen with the message "Assertion Failed", just that.

I'm trying to use it on the Rootstook RSK network, with a Ledger Nano X. The failure just appears if using the Desktop version, on a system where MS Edge is installed as the default browser. When using the app.mycrypto.com web app, with MS EDGE, the "Select an account" page hangs too ... but it works ok on Chrome!

It worked a couple of weeks ago.

Here's the assertion details:

vendors~client~vendor.bf10d6082ae688e1f030.js:73 Error: Assertion failed
    at n (vendors~client~vendor.bf10d6082ae688e1f030.js:1)
    at a.imuln (vendors~client~vendor.bf10d6082ae688e1f030.js:1)
    at a.muln (vendors~client~vendor.bf10d6082ae688e1f030.js:1)
    at equivalentValues.Object.keys.map.t (client.c69bad17cdba29b05016.js:1)
    at Array.map (<anonymous>)
    at Object.values.map.map (client.c69bad17cdba29b05016.js:1)
    at Array.map (<anonymous>)
    at f.handleAllValues.e [as handleAllValues] (client.c69bad17cdba29b05016.js:1)
    at f.generateValues (client.c69bad17cdba29b05016.js:1)
    at f.render (client.c69bad17cdba29b05016.js:1)

I've tested it with having EDGE as the default windows browser, and with Chrome too. Same error.

same issue here

vendors~client~vendor.bf10d6082ae688e1f030.js:73 Error: Assertion failed
    at n (vendors~client~vendor.bf10d6082ae688e1f030.js:1)
    at a.imuln (vendors~client~vendor.bf10d6082ae688e1f030.js:1)
    at a.muln (vendors~client~vendor.bf10d6082ae688e1f030.js:1)
    at equivalentValues.Object.keys.map.t (client.c69bad17cdba29b05016.js:1)
    at Array.map (<anonymous>)
    at Object.values.map.map (client.c69bad17cdba29b05016.js:1)
    at Array.map (<anonymous>)
    at f.handleAllValues.e [as handleAllValues] (client.c69bad17cdba29b05016.js:1)
    at f.generateValues (client.c69bad17cdba29b05016.js:1)
    at f.render (client.c69bad17cdba29b05016.js:1)
aa @ vendors~client~vendor.bf10d6082ae688e1f030.js:73

--- a/common/components/BalanceSidebar/EquivalentValues.tsx
+++ b/common/components/BalanceSidebar/EquivalentValues.tsx
@@ -255,7 +255,7 @@ class EquivalentValues extends React.Component<Props, State> {
           ...Object.keys(rateType).map(rate => {
             const balanceIndex: AllValue = balance[i];
             const value =
-              balanceIndex && !!balanceIndex.balance
+              balanceIndex && !!balanceIndex.balance && rateType[rate] < 0x4000000
                 ? balanceIndex.balance.muln(rateType[rate])
                 : null;
             return { rate, value };

